misc/p.c

```diff
@@ -0,0 +1,25 @@
+#include<stdio.h>
+#include<conio.h>
```

Comment:
please avoid using ``conio.h`` _(getch)_, as it is platform specific, & wouldn't compile on linux _(gcc)_ & many other compilers.